collections/src/main/scala/strawman/collection/immutable/LazyList.scala

```diff
@@ -453,14 +453,12 @@ object LazyList extends LazyListFactory[LazyList] {
 
   protected[this] def newCons[T](hd: => T, tl: => LazyList[T]): LazyList[T] = new LazyList.Cons(hd, tl)
 
-  type Evaluated[+A] = Option[(A, LazyList[A])]
-
   object Empty extends LazyList[Nothing] {
     override def isEmpty: Boolean = true
     override def head: Nothing = throw new NoSuchElementException("head of empty lazy list")
     override def tail: LazyList[Nothing] = throw new UnsupportedOperationException("tail of empty lazy list")
-    def force: Evaluated[Nothing] = None
-    override def toString: String = "Empty"
+    override def force: LazyList[Nothing] = this
```

Comment:
I prefer you to omit `override` when you only *implement*

Reply:
Fixed.